static/css/style.css

```diff
@@ -0,0 +1,77 @@
+:root {
```

Comment:
where does this initial startup come from? do you have a template saved somewhere?

Reply:
The `:root` is based on a blogger who does good CSS. I don't have a template saved anywhere. I usually just go to a previous project then cut and paste.

Reply:
maybe setup https://github.com/katlukens/templates with this?